assets/quick-start-module.js

```diff
@@ -243,7 +243,13 @@ function commandMessage(key) {
       "pip install torch==1.8.0+cu111 torchvision==0.9.0+cu111 torchaudio===0.8.0 -f https://download.pytorch.org/whl/torch_stable.html",
 
     "stable,pip,windows,rocm4.0,python":
-      "<b>NOTE:</b> ROCm is not available on Windows<br />",
+      "<b>NOTE:</b> ROCm is not available on Windows",
+
+    "stable,source,macos,rocm4.0,python":
+      "<b>NOTE:</b> ROCm is not available on MacOS",
+
+    "stable,source,windows,rocm4.0,python":
+       "<b>NOTE:</b> ROCm is not available on Windows",
```

Comment:
All build from source requests should just redirect to github.
```suggestion
```

Reply:
Done.